spec/javascript/e2e/FlaggedApplicationsShowPage.e2e.js

```diff
@@ -38,7 +38,7 @@ describe('FlaggedApplicationsShowPage', () => {
       await saveButtonList[0].click()
 
       // Expect the row to be closed
-      await page.waitFor(() => !document.querySelector('.rt-expander.-open'))
+      await page.waitForFunction(() => !document.querySelector('.rt-expander.-open'))
```

Comment:
waitFor -> waitForFunction and waitForTimeout turned out to be the only breaking-ish change from upgrading puppeteer